dwitter/static/libs/loopstopper.js

```diff
@@ -0,0 +1,43 @@
+window.stopper = {
+
+  loops: [],
+  loopTimers: [],
+  TIME_OUT_LENGTH: 500,
+
+  // States:
```

Comment:
Is there a significant performance penalty to use some sort of enum here to increase readability? Not a big deal as there isn't too much code, and it is quite performance critical